firefox_services_tests/helpers/__init__.py

```diff
@@ -0,0 +1 @@
+__author__ = 'jdorlus'
```

Comment:
We don't need this here however we do need a licence. same for all `__init__.py`